SWLOR.Game.Server/Feature/LootTableDefinition/ViscaraLootTableDefinition.cs

```diff
@@ -191,6 +207,21 @@ private void VellenFlesheater()
                 .AddItem("babonsch", 5, 1, true);
         }
 
+        private void CoxxianCrate()
```

Comment:
And one more here. FYI - it's inconsistent because I made the same mistake but the official spelling is "Coxxion", no 'A'. Really has no impact so feel free to leave the name as-is if you want. Please move this to the resources class for better organization